### PR TITLE
fix access token next button

### PR DESCRIPTION
**Pull request by Leandroqo**
## Issue Description

It is currently possible to proceed by clicking or not on the "Next" button.

## Solution

unlock the next steps by clicking the next button or pressing enter.

### CODE REVIEW

**Review by bCatanant — APPROVED** [review]

File: ui/src/modules/Tokens/Comparation/__tests__/Tab.spec.tsx [hunk 1]
```diff
@@ -26,33 +26,48 @@ test('render Token View mode', async () => {
...
-        "workspaces": "",
-        "allWorkspaces": true,
-        "revoked": false,
-        "created_at": "2021-04-12T23:02:39.304307Z",
-        "revoked_at": "",
-        "last_used_at": "",
-        "author":"charlesadmin@admin"
-      }))
-    }),
+    rest.get(
```
Comment (luanecavalcantizup) [inline ui/src/modules/Tokens/Comparation/__tests__/Tab.spec.tsx:29]: why did you change from server.use() to rest.get()?

**Review by luanecavalcantizup — APPROVED** [review]
